MAINTAINERS.md

```diff
@@ -30,19 +30,19 @@ If you are an owner of the organization, you should be able to see an automated
 - **Lucas Schneider ([@schneiderl](https://github.com/schneiderl))**:
   [11 April 2019](https://github.com/tldr-pages/tldr/issues/2898) — present
 - **Andrik Albuquerque ([@andrik](https://github.com/andrik))**:
-  [9 May 2019](https://github.com/tldr-pages/tldr/issues/2988) - present
+  [9 May 2019](https://github.com/tldr-pages/tldr/issues/2988) — present
```

Comment:
We should keep the ASCII dashes here I think, @zdroid. This would be consistent with our existing approach to the pages themselves, where we prefer ASCII quotes instead of smart ones, for instance.

Reply:
This file isn't supposed to be viewed as plain text, but as processed Markdown. It's also provided the file will be viewed on GitHub's web interface, which supports this style. Also, it's not appropriate to use hyphens for this kind of indication.

Reply:
I think it's ok to use em dashes here. I didn't notice them from the raw text, but it makes a difference when viewing the rendered document, em dashes look way better IMHO.

Reply:
Hrm, I guess that makes sense. This is still markdown, it's just a case of being dependent on where it's going to be viewed.